package ExceptionTest;

import java.io.IOException;
import java.util.concurrent.ExecutionException;

public class ExceptionTest {
    public static void main(String[] args){
        /*try{
            System.out.println("Ganesh");
        }
        catch (IOException e){  // compile time error as IOException is not being thrown

        }*/
    }
}
